Vote Text for comments (#8)
Valdorff/rp-thoughts: vote_text_drafts/2023_11_budget_update.md

> Vote options:
> - For
> - Against
> - Abstain

> # 2023-11 Budget Update (RPIP-10)

Reviewer: General structure seems good.

This vote aims to correct a budget implementation error and further increase the IMC budget.

> ### Outcomes

Reviewer: Need a note that these are the outcomes if there is a positive result.

> - Correct the mismatch between intent and implementation in an [earlier vote](https://vote.rocketpool.net/#/proposal/0x510383ca82a0096fa670a260692cf7a4097e199ce4f731dc4efd97a21f19f988)

Reviewer: ```suggestion
- The mismatch between intent and implementation in an [earlier vote](https://vote.rocketpool.net/#/proposal/0x510383ca82a0096fa670a260692cf7a4097e199ce4f731dc4efd97a21f19f988) will be corrected.
```

>   - One-time payment to make up for not matching intent for several periods

Reviewer: ```suggestion
  - A one-time payment will be made to the IMC from the pDAO treasury to make up for the periods since the mismatch. 
```

>   - Correct the split to match intent moving forward for both IMC and GMC

Reviewer: I think you can just remove this bullet? The intent changing is covered by the IMC share increase. Otherwise its just matching implementation to the previous intent.

Author: It's hierarchical - we're correcting to the overall intent by (A) updating split going forward and (B) a one-time payment to make up for the intervening period. I'm going to make it more obviously like that and adapt most of your more formal wording

> - Further increase the IMC’s split by 16%

Reviewer: ```suggestion
- The IMC's split of the pDAO inflation budget will be increased by 16%
```
Need to be a little careful around the use of percentages. Is this 16% of the total inflation, 16% of the old IMC share? etc

Author: Yep - good call. Going with "of the pDAO's incoming funds" to match wording in RPIP-10

### Rationale

> The earlier vote had a math error in the implementation (as noted on the [forum](https://dao.rocketpool.net/t/intended-vs-actual-budget-mismatch-in-rpip-10/2053?u=valdorff)). This vote undoes that both moving forward and retroactively via a one-time transfer.

Reviewer: ```suggestion
The earlier vote had a math error in the implementation (as noted on the [forum](https://dao.rocketpool.net/t/intended-vs-actual-budget-mismatch-in-rpip-10/2053?u=valdorff)). This vote  is to resolve the error both moving forward and retroactively via a one-time transfer.
```

> In addition, this vote increases the IMC's portion of the inflation split by 16%, which is roughly "50% of funds that were initially going to the oDAO that now go to the pDAO". This value was arrived at mostly from a [forum poll](https://dao.rocketpool.net/t/justification-for-moving-odao-inflation-to-the-imc/1939/12), but taking a conservative step as a starting point. The IMC will gain additional freedom, and there's been significant discussion around using some of that freedom to provide more RPL liquidity.  

Reviewer: ```suggestion
In addition, this vote increases the IMC's portion of the inflation split by 16%, which is roughly "50% of funds that were initially going to the oDAO that now go to the pDAO". This value was arrived at via this [forum poll](https://dao.rocketpool.net/t/justification-for-moving-odao-inflation-to-the-imc/1939/12), but taking a conservative step as a starting point. The IMC will gain additional freedom, and there's been significant discussion around using some of that freedom to provide more RPL liquidity.  
```

The sentence with the 'conservative step' doesn't read well, and I'm not sure what you're trying to say. Consider rewording that too. 

The point around additional freedom is also slightly out of left field. This is freedom as a result of having more funds rather than a relaxation of the controls on the IMCs spending, right? If it is a relaxation of the controls, this should be highlighted in outcomes. If it isn't, then this should be reworded to avoid giving this impression.

Author: This RPIP does not go with the most popular sentiment in the poll. Of the popular options, it chooses the smallest (conservative) step.

I will nuke the "additional freedom" sentence.
